packages/modal/src/Modal/Modal.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import type { Meta } from "@storybook/react";
+import { Button } from "@telegraph/button";
+import { Heading } from "@telegraph/typography";
+
+import { Modal as TelegraphModal } from "./Modal";
+
+const meta: Meta<typeof TelegraphModal> = {
+  title: "Components/Modal",
+  // component: TelegraphModal,
+};
+export default meta;
+
+export const Modal = {
+  render: () => (
+    <TelegraphModal.Root open={true}>
```

Comment:
Are you getting typesense for these props? Testing locally with the story, I'm only seeing `key?` and `ref?` as props. Not a blocking issue, but it'd be great to have that so these components are easier to work with in Control.

Reply:
Because it's not a "Default" component and rather the `Modal.ComponentPart` pattern it doesn't work great with storyboook. Once we decide what a good default it is we'll swap that in here and the type sense will be better.